Disable the outfile_loaddata test, which as of 5.1.38 is giving
platform-dependent results, with the "expected" results being arguably the
wrong ones.  This is upstream at http://bugs.mysql.com/bug.php?id=46895
(note that upstream has also disabled it, but only for Solaris, so we still
need this patch).
Still broken in 5.5.14, despite alleged fix.

Also disable sys_vars.plugin_dir_basic, which is broken because
mysql-test-run doesn't set the right value of MYSQL_LIBDIR.  Upstream at
http://bugs.mysql.com/bug.php?id=52223
Still fails in 5.5.14, even though upstream marked it closed.

Also disable innodb.innodb, which is showing platform-dependent results
as of 5.5.9.  Upstream at http://bugs.mysql.com/bug.php?id=60155


diff -Naur mysql-5.5.14.orig/mysql-test/t/disabled.def mysql-5.5.14/mysql-test/t/disabled.def
--- mysql-5.5.14.orig/mysql-test/t/disabled.def	2011-06-21 12:42:40.000000000 -0400
+++ mysql-5.5.14/mysql-test/t/disabled.def	2011-07-11 14:33:46.560138525 -0400
@@ -17,3 +17,7 @@
 archive-big              : Bug#11817185 2011-03-10 Anitha Disabled since this leads to timeout on Solaris Sparc
 main.query_cache_28249   : Bug#12584161 2009-03-25 main.query_cache_28249 fails sporadically
 log_tables-big           : Bug#11756699 2010-11-15 mattiasj report already exists
+#
+outfile_loaddata         : bug#46895 code wrong, expected results wrong too
+sys_vars.plugin_dir_basic : bug#52223 fails for lib64 library directory
+innodb.innodb            : bug#60155 has platform-dependent results
